tools/IoTEdgeDevOps/VstsPipelineSync/db/dbo.UpsertVstsRelease.sql

```diff
@@ -16,10 +17,18 @@ AS
 	DECLARE @now datetime2;
 	SET @now = SYSDATETIME();
 
+    exec [dbo].[CheckForNull] @i = @Id
+    exec [dbo].[CheckForNull] @i = @SourceBranch
+    exec [dbo].[CheckForNull] @i = @Status
+    exec [dbo].[CheckForNull] @i = @WebUri
+    exec [dbo].[CheckForNull] @i = @DefinitionId
+    exec [dbo].[CheckForNull] @i = @DefinitionName
```

Comment:
I altered the table to add another column for source branch, but couldn't create a constraint of NOT NULL because of existing records. This is a hack because I didn't want any jobs running populating the tables with a subset of required data. I should drop the release tables, recreate with constraints, and get rid of these null checks.